core/src/main/res/layout/noplayer_view.xml

```diff
@@ -1,5 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
-<com.google.android.exoplayer2.ui.SimpleExoPlayerView xmlns:android="http://schemas.android.com/apk/res/android"
-  android:id="@+id/simple_player_view"
+<com.google.android.exoplayer2.ui.AspectRatioFrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  android:id="@+id/video_frame"
   android:layout_width="match_parent"
-  android:layout_height="wrap_content" />
+  android:layout_height="wrap_content">
+
+  <SurfaceView
+    android:id="@+id/surface_view"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_gravity="center" />
+
+  <com.google.android.exoplayer2.ui.SubtitleView
```

Comment:
I'm happy with it as is for now. When we get some time out of the current changes we are doing we can implement a no-player version 😄